addons/core/tests/integration/helpers/relative-datetime-live-test.js

```diff
@@ -35,7 +35,7 @@ module('Integration | Helper | relative-datetime-live', function (hooks) {
     await render(hbs`{{relative-datetime-live this.date}}`);
     assert.dom(this.element).hasText('23 years ago');
 
-    this.date = now + 8 * 31 * 24 * 60 * 60 * 1000;
+    this.date = now + 8 * 30 * 24 * 60 * 60 * 1000;
```

Comment:
I noticed this test was failing for me as well, but when I reran it, it passed 🤯

Reply:
@DhariniJeeva I think it may be related to the time change